zebra-scan/src/tests.rs

```diff
@@ -550,3 +509,45 @@ fn random_compact_tx(mut rng: impl RngCore) -> CompactTx {
     ctx.outputs.push(cout);
     ctx
 }
+
+/// Returns transactions belonging to any of the given [`ScanningKey`]s.
+///
+/// TODO:
+/// - Remove the `sapling_tree_size` parameter or turn it into an `Option` once we have access to
+/// Zebra's state, and we can retrieve the tree size ourselves.
+/// - Add prior block metadata once we have access to Zebra's state.
+fn scan_block<K: ScanningKey>(
```

Comment:
I'm going to move the function to the new file.

Reply:
Done in [f49e936](https://github.com/ZcashFoundation/zebra/pull/7994/commits/f49e936b6b3c9ee5c761edb3d2d67a6976b282b5).